mcmd/commands/delete.py

```diff
@@ -81,6 +92,17 @@ def _delete_entity_type_data(args):
         client.delete(url)
 
 
+def _delete_entity_type_attribute(args):
+    if args.force or (not args.force and io.confirm(
+            'Are you sure you want to delete attribute {} of entity type {}?'.format(args.attribute, args.resource))):
+        io.start('Deleting attribute {} of entity {}'.format(highlight(args.attribute), highlight(args.resource)))
+        response = client.get(urljoin(config.api('rest2'),
```

Comment:
encoding issues (see comments in #118)

Reply:
Encoding will be done in the `api` module as soon as #118 is merged